packages/suite/src/config/wallet/networks.ts

```diff
@@ -291,7 +291,7 @@ const networks = {
             tx: 'https://ropsten1.trezor.io/tx/',
             account: 'https://ropsten1.trezor.io/address/',
         },
-        features: ['sign-verify'],
+        features: ['sign-verify', 'tokens'],
```

Comment:
```suggestion
        features: ['sign-verify', 'tokens', 'amount-unit'],
```
missing feature for regtest

Reply:
still missing

Reply:
The line you referenced belongs to Ethereum Ropsten, I don’t quiet get what the issue is 🤔 The feature is there for Bitcoin Regtest.

Reply:
sorry i've missed that even if i double checked before writing it, i guess im blind